prime-router/src/main/kotlin/ReportStreamFilterDefinition.kt

```diff
@@ -396,6 +396,33 @@ class HasAtLeastOneOf : ReportStreamFilterDefinition {
     }
 }
 
+/**
+ * intersect("condition1"|"condition2"|"condition3"|)
+ * Implements a condition check match.  If a row has valid data for any of the conditions, the row is selected.
+ */
+class Intersect : ReportStreamFilterDefinition {
```

Comment:
Remove

Reply:
See this comment: https://github.com/CDCgov/prime-reportstream/pull/8315#discussion_r1110373333